company.el

```diff
@@ -1091,7 +1091,9 @@ matches IDLE-BEGIN-AFTER-RE, return it wrapped in a cons."
 
 (defvar-local company-common nil)
 
-(defvar-local company-selection 0)
+(defvar company-selection-default 0
```

Comment:
We could also make this a boolean, since there are only two valid values here, really. That would make initializing `company-selection` just a little bit more complicated.

Reply:
Yes, but I think we can put another PR for that. Doing that now will make this PR more complicated.

Reply:
Ok.